### DISCUSSION TITLE
How to build Paraview on macOS

### DISCUSSION THREAD

**Discussion by earlsioson** [Kitware/trame#312] [Trame/ParaView]
@jourdain, how would I build paraview on macOS? I tried installing python3.9-dev via pyenv but pyenv clones version 3.9.17 which causes this error...

Could NOT find Python3 (missing: Development.Module) (found suitable version "3.9.17", minimum required is "3.4")

here is the build command I used...

cmake
-S ~/dev/install/pv-dev/src/ParaView
-B ~/dev/install/pv-dev/build
-G Ninja
-D PARAVIEW_USE_PYTHON=ON
-D PARAVIEW_USE_MPI=OFF
-D VTK_OPENGL_HAS_EGL=OFF
-D VTK_OPENGL_HAS_OSMESA=OFF
-D VTK_USE_X=OFF
-D VTK_DEFAULT_RENDER_WINDOW_HEADLESS=ON
-D CMAKE_BUILD_TYPE=Release
-D PARAVIEW_USE_QT=OFF
-DPython3_EXECUTABLE=/Users/earl/.pyenv/versions/3.9-dev/bin/python3
-DPython3_INCLUDE_DIRS=/Users/earl/.pyenv/versions/3.9-dev/include
-DPython3_LIBRARY=/Users/earl/.pyenv/versions/3.9-dev/lib/libpython3.9.dylib
-DPython3_INCLUDE_DIR=/Users/earl/.pyenv/versions/3.9-dev/include
-DPython3_LIBRARY_DIRS=/Users/earl/.pyenv/versions/3.9-dev/lib
&& cmake --build ~/dev/install/pv-dev/build
&& cmake --install ~/dev/install/pv-dev/build --prefix ~/dev/install/paraview

**Comment by jourdain** [comment]
Might be better to ask that question on https://discourse.paraview.org/

**Reply by earlsioson** [reply]
Thanks @jourdain, will do.
